packages/jellyfish-block/__tests__/rpc/blockHeader.test.ts

```diff
@@ -0,0 +1,34 @@
+import { MasterNodeRegTestContainer } from '@defichain/testcontainers'
+import { Testing } from '@defichain/jellyfish-testing'
+import { SmartBuffer } from 'smart-buffer'
+import { CBlockHeader } from '../../src/blockHeader'
+import BigNumber from 'bignumber.js'
+
+describe('BlockHeader', () => {
+  const testing = Testing.create(new MasterNodeRegTestContainer())
+
+  beforeAll(async () => {
+    await testing.container.start()
+  })
+
+  afterAll(async () => {
+    await testing.container.stop()
+  })
+
+  it('should getBlockHeader with verbosity=true and have same data as CBlockHeader composer to object', async () => {
+    const blockHash = await testing.misc.waitForBlockHash(1)
+    const hexData = await testing.rpc.blockchain.getBlockHeader(blockHash, false)
+    const buffer = SmartBuffer.fromBuffer(Buffer.from(hexData, 'hex'))
+    const composable = new CBlockHeader(buffer)
+
+    const blockHeader = await testing.rpc.blockchain.getBlockHeader(blockHash, true)
```

Comment:
`getblockheader` RPC does not return `mintedBlocks` and `signature`. should it return those as well? if then log a ticket to c++ side.

Reply:
https://github.com/DeFiCh/ain/issues/737